Add new workflow sample as a new Box

kohki1234 opened the pull request:
I added a new workflow sample which kicks master segment workflow, then calls profile API token workflow

mickeey2525 commented:
Basically, looks good to me.
I have a concern about task name. `+call_master_segment`
For me, `call` sounds like `call` operator. 
So how about using invoke or something?

kohki1234 commented:
Thank you for the feedback, I agree with your idea and made 2 changes.

- changed the task name to `+invoke_master_segment`
- changed the task description to avoid confusion.
# require audience workflow if there is no running attempt
